DataFormats/SiPixelDigiSoA/interface/SiPixelDigiErrorsHost.h

```diff
@@ -22,8 +22,8 @@ class SiPixelDigiErrorsHost : public PortableHostCollection<SiPixelDigiErrorsSoA
 
   int maxFedWords() const { return maxFedWords_; }
 
-  auto& error_data() { return (*view().pixelErrors()); }
-  auto const& error_data() const { return (*view().pixelErrors()); }
+  auto& error_data() { return view()[0].pixelErrors(); }
+  auto const& error_data() const { return view()[0].pixelErrors(); }
```

Comment:
Mhm, I find this syntax rather confusing.

In fact, I'm wondering that returning an `std::span<T>` for a scalar may not be the best approach ?

What would be the implications of returning a raw pointer `T*`, or returning by reference `T&`, or returning an `std::reference_wrapper<T>` ?

Reply:
Ah OK, thanks for the clarification and sorry for the noise 🤷🏻‍♂️

Reply:
OK, then I'm still confused, by both original code as well as the new one...

However from a quick check it looks like `error_data()` is never used 🤔 
Could you try to just remove these methods and see if anything breaks ?